driver/bpf/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM debian:unstable
+
+MAINTAINER Sysdig <support@sysdig.com>
+
+ENV SYSDIG_HOST_ROOT /host
+
+ENV AGENT_VERSION 0.1.1dev
+
+ENV SYSDIG_SRC /draios
+
+ENV HOME /root
+
+ENV CXX /usr/bin/g++
+
+# Based on the sysdig container, used for building eBPF probe
+
+RUN apt-get update \
+ && apt-get dist-upgrade -y \
+ && apt-get install -y --no-install-recommends \
+    apt-utils \
+    cmake \
```

Comment:
What is the logic behind tweaking this dependency list? It's certainly ok to do so, since this container will never be distributed to a customer, but I just wanted to understand the rationale.

In particular, it seems like you added some packages that are not needed to compile the bpf probe (e.g. `cmake`, `g++`), removed some packages that are not needed to compile the bpf probe (e.g. `dkms`) and kept some other packages that are also not needed to compile the bpf probe (e.g. everything else besides `make`, `gcc`, `llvm` and `clang`). Is it because you envision this container with the ability of doing more than just building the probe or something else?

Reply:
My original thought was to have this container build both the kmod and the probe. I decided to simplify, but didn't remove all the extra dependencies. I'll remove them.